pyccl/ccl_p2d.i

```diff
@@ -0,0 +1,52 @@
+%module ccl_p2d
+
+%{
+/* put additional #include here */
+%}
+
+// Enable vectorised arguments for arrays
+%apply (double* IN_ARRAY1, int DIM1) {(double* lkarr, int nk)};
+%apply (double* IN_ARRAY1, int DIM1) {(double* aarr, int na)};
+%apply (double* IN_ARRAY1, int DIM1) {(double* pkarr, int npk)};
+%apply (int DIM1, double* ARGOUT_ARRAY1) {(int ndout, double* doutput)};
+
+%include "../include/ccl_p2d.h"
+%include "../include/ccl_core.h"
+
+%inline %{
+ccl_p2d_t *set_p2d_new_from_arrays(double* lkarr,int nk,
+				   double* aarr,int na,
+				   double* pkarr,int npk,
+				   int order_lok,int order_hik,
+				   int is_logp,
+				   int *status)
+{
+  ccl_p2d_t *psp=ccl_p2d_t_new(na,aarr,nk,lkarr,pkarr,
+			       1,2,ccl_p2d_cclgrowth,
```

Comment:
You didn't pass the extrapolation orders here.

Reply:
Argh, stupid me. Fixed